testing/tests/DevExpress.ui.widgets.dataGrid/grouping.integration.tests.js

```diff
@@ -227,38 +227,6 @@ QUnit.module('Initialization', baseModuleConfig, () => {
         assert.equal(dataGrid.columnOption('field2', 'groupIndex'), undefined, 'field2 has no groupIndex');
     });
 
-    QUnit.test('Group panel should set correct \'max-width\' after clear grouping', function(assert) {
```

Comment:
Because in _checkSize_ method we check grid's visibility, we need to move this test to testcafe